2011-08-12 23:04:49,366 WARN org.apache.hadoop.util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
2011-08-12 23:04:49,637 INFO org.apache.hadoop.metrics.jvm.JvmMetrics: Initializing JVM Metrics with processName=SHUFFLE, sessionId=
2011-08-12 23:04:49,850 INFO org.apache.hadoop.mapred.ReduceTask: ShuffleRamManager: MemoryLimit=144965632, MaxSingleShuffleLimit=36241408
2011-08-12 23:04:49,858 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Thread started: Thread for merging on-disk files
2011-08-12 23:04:49,858 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Thread waiting: Thread for merging on-disk files
2011-08-12 23:04:49,858 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Thread started: Thread for merging in memory files
2011-08-12 23:04:49,860 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Thread started: Thread for polling Map Completion Events
2011-08-12 23:04:49,861 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:04:49,862 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (0 slow hosts and0 dup hosts)
2011-08-12 23:04:49,865 INFO org.apache.hadoop.mapred.ReduceTask: Ignoring obsolete output of FAILED map-task: 'attempt_201108121808_0005_m_000000_0'
2011-08-12 23:04:49,865 INFO org.apache.hadoop.mapred.ReduceTask: Ignoring obsolete output of FAILED map-task: 'attempt_201108121808_0005_m_000001_0'
2011-08-12 23:04:54,863 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 1 outputs (0 slow hosts and0 dup hosts)
2011-08-12 23:04:55,233 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 copy failed: attempt_201108121808_0005_m_000000_1 from nothing
2011-08-12 23:04:55,235 WARN org.apache.hadoop.mapred.ReduceTask: java.net.UnknownHostException: nothing
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:195)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:432)
	at java.net.Socket.connect(Socket.java:529)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:158)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:394)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:529)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:233)
	at sun.net.www.http.HttpClient.New(HttpClient.java:306)
	at sun.net.www.http.HttpClient.New(HttpClient.java:323)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:970)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:911)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:836)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getInputStream(ReduceTask.java:1526)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.setupSecureConnection(ReduceTask.java:1483)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getMapOutput(ReduceTask.java:1391)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.copyOutput(ReduceTask.java:1302)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.run(ReduceTask.java:1234)

2011-08-12 23:04:55,236 INFO org.apache.hadoop.mapred.ReduceTask: Task attempt_201108121808_0005_r_000000_2: Failed fetch #1 from attempt_201108121808_0005_m_000000_1
2011-08-12 23:04:55,236 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 adding host nothing to penalty box, next contact in 4 seconds
2011-08-12 23:04:55,236 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2: Got 1 map-outputs from previous failures
2011-08-12 23:05:00,236 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 1 outputs (0 slow hosts and0 dup hosts)
2011-08-12 23:05:00,295 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 copy failed: attempt_201108121808_0005_m_000000_1 from nothing
2011-08-12 23:05:00,295 WARN org.apache.hadoop.mapred.ReduceTask: java.net.UnknownHostException: nothing
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:195)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:432)
	at java.net.Socket.connect(Socket.java:529)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:158)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:394)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:529)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:233)
	at sun.net.www.http.HttpClient.New(HttpClient.java:306)
	at sun.net.www.http.HttpClient.New(HttpClient.java:323)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:970)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:911)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:836)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getInputStream(ReduceTask.java:1526)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.setupSecureConnection(ReduceTask.java:1483)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getMapOutput(ReduceTask.java:1391)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.copyOutput(ReduceTask.java:1302)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.run(ReduceTask.java:1234)

2011-08-12 23:05:00,296 INFO org.apache.hadoop.mapred.ReduceTask: Task attempt_201108121808_0005_r_000000_2: Failed fetch #2 from attempt_201108121808_0005_m_000000_1
2011-08-12 23:05:00,296 INFO org.apache.hadoop.mapred.ReduceTask: Failed to fetch map-output from attempt_201108121808_0005_m_000000_1 even after MAX_FETCH_RETRIES_PER_MAP retries...  or it is a read error,  reporting to the JobTracker
2011-08-12 23:05:00,296 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 adding host nothing to penalty box, next contact in 8 seconds
2011-08-12 23:05:00,296 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2: Got 1 map-outputs from previous failures
2011-08-12 23:05:10,297 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 1 outputs (0 slow hosts and0 dup hosts)
2011-08-12 23:05:10,305 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 copy failed: attempt_201108121808_0005_m_000000_1 from nothing
2011-08-12 23:05:10,306 WARN org.apache.hadoop.mapred.ReduceTask: java.net.UnknownHostException: nothing
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:195)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:432)
	at java.net.Socket.connect(Socket.java:529)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:158)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:394)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:529)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:233)
	at sun.net.www.http.HttpClient.New(HttpClient.java:306)
	at sun.net.www.http.HttpClient.New(HttpClient.java:323)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:970)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:911)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:836)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getInputStream(ReduceTask.java:1526)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.setupSecureConnection(ReduceTask.java:1483)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getMapOutput(ReduceTask.java:1391)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.copyOutput(ReduceTask.java:1302)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.run(ReduceTask.java:1234)

2011-08-12 23:05:10,306 INFO org.apache.hadoop.mapred.ReduceTask: Task attempt_201108121808_0005_r_000000_2: Failed fetch #3 from attempt_201108121808_0005_m_000000_1
2011-08-12 23:05:10,326 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 adding host nothing to penalty box, next contact in 150 seconds
2011-08-12 23:05:10,327 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2: Got 1 map-outputs from previous failures
2011-08-12 23:05:50,329 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:05:50,330 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (1 slow hosts and0 dup hosts)
2011-08-12 23:05:50,330 INFO org.apache.hadoop.mapred.ReduceTask: Penalized(slow) Hosts: 
2011-08-12 23:05:50,330 INFO org.apache.hadoop.mapred.ReduceTask: nothing Will be considered after: 109 seconds.
2011-08-12 23:06:50,334 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:06:50,335 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (1 slow hosts and0 dup hosts)
2011-08-12 23:06:50,335 INFO org.apache.hadoop.mapred.ReduceTask: Penalized(slow) Hosts: 
2011-08-12 23:06:50,335 INFO org.apache.hadoop.mapred.ReduceTask: nothing Will be considered after: 49 seconds.
2011-08-12 23:07:40,339 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 1 outputs (0 slow hosts and0 dup hosts)
2011-08-12 23:07:40,346 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 copy failed: attempt_201108121808_0005_m_000000_1 from nothing
2011-08-12 23:07:40,346 WARN org.apache.hadoop.mapred.ReduceTask: java.net.UnknownHostException: nothing
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:195)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:432)
	at java.net.Socket.connect(Socket.java:529)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:158)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:394)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:529)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:233)
	at sun.net.www.http.HttpClient.New(HttpClient.java:306)
	at sun.net.www.http.HttpClient.New(HttpClient.java:323)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:970)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:911)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:836)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getInputStream(ReduceTask.java:1526)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.setupSecureConnection(ReduceTask.java:1483)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getMapOutput(ReduceTask.java:1391)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.copyOutput(ReduceTask.java:1302)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.run(ReduceTask.java:1234)

2011-08-12 23:07:40,346 INFO org.apache.hadoop.mapred.ReduceTask: Task attempt_201108121808_0005_r_000000_2: Failed fetch #4 from attempt_201108121808_0005_m_000000_1
2011-08-12 23:07:40,346 INFO org.apache.hadoop.mapred.ReduceTask: Failed to fetch map-output from attempt_201108121808_0005_m_000000_1 even after MAX_FETCH_RETRIES_PER_MAP retries...  or it is a read error,  reporting to the JobTracker
2011-08-12 23:07:40,346 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 adding host nothing to penalty box, next contact in 150 seconds
2011-08-12 23:07:40,347 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2: Got 1 map-outputs from previous failures
2011-08-12 23:07:50,350 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:07:50,351 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (1 slow hosts and0 dup hosts)
2011-08-12 23:07:50,351 INFO org.apache.hadoop.mapred.ReduceTask: Penalized(slow) Hosts: 
2011-08-12 23:07:50,352 INFO org.apache.hadoop.mapred.ReduceTask: nothing Will be considered after: 139 seconds.
2011-08-12 23:08:50,356 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:08:50,356 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (1 slow hosts and0 dup hosts)
2011-08-12 23:08:50,356 INFO org.apache.hadoop.mapred.ReduceTask: Penalized(slow) Hosts: 
2011-08-12 23:08:50,356 INFO org.apache.hadoop.mapred.ReduceTask: nothing Will be considered after: 79 seconds.
2011-08-12 23:09:50,359 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:09:50,360 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (1 slow hosts and0 dup hosts)
2011-08-12 23:09:50,360 INFO org.apache.hadoop.mapred.ReduceTask: Penalized(slow) Hosts: 
2011-08-12 23:09:50,360 INFO org.apache.hadoop.mapred.ReduceTask: nothing Will be considered after: 19 seconds.
2011-08-12 23:10:10,362 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 1 outputs (0 slow hosts and0 dup hosts)
2011-08-12 23:10:10,366 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 copy failed: attempt_201108121808_0005_m_000000_1 from nothing
2011-08-12 23:10:10,368 WARN org.apache.hadoop.mapred.ReduceTask: java.net.UnknownHostException: nothing
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:195)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:432)
	at java.net.Socket.connect(Socket.java:529)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:158)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:394)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:529)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:233)
	at sun.net.www.http.HttpClient.New(HttpClient.java:306)
	at sun.net.www.http.HttpClient.New(HttpClient.java:323)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:970)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:911)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:836)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getInputStream(ReduceTask.java:1526)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.setupSecureConnection(ReduceTask.java:1483)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.getMapOutput(ReduceTask.java:1391)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.copyOutput(ReduceTask.java:1302)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$MapOutputCopier.run(ReduceTask.java:1234)

2011-08-12 23:10:10,368 INFO org.apache.hadoop.mapred.ReduceTask: Task attempt_201108121808_0005_r_000000_2: Failed fetch #5 from attempt_201108121808_0005_m_000000_1
2011-08-12 23:10:10,368 WARN org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 adding host nothing to penalty box, next contact in 150 seconds
2011-08-12 23:10:10,368 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2: Got 1 map-outputs from previous failures
2011-08-12 23:10:50,371 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:10:50,371 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (1 slow hosts and0 dup hosts)
2011-08-12 23:10:50,371 INFO org.apache.hadoop.mapred.ReduceTask: Penalized(slow) Hosts: 
2011-08-12 23:10:50,371 INFO org.apache.hadoop.mapred.ReduceTask: nothing Will be considered after: 109 seconds.
2011-08-12 23:11:41,098 INFO org.apache.hadoop.mapred.Task: Communication exception: java.io.IOException: Call to /127.0.0.1:49442 failed on local exception: java.io.EOFException
	at org.apache.hadoop.ipc.Client.wrapException(Client.java:1142)
	at org.apache.hadoop.ipc.Client.call(Client.java:1110)
	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:226)
	at $Proxy0.statusUpdate(Unknown Source)
	at org.apache.hadoop.mapred.Task$TaskReporter.run(Task.java:635)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:375)
	at org.apache.hadoop.ipc.Client$Connection.receiveResponse(Client.java:815)
	at org.apache.hadoop.ipc.Client$Connection.run(Client.java:724)

2011-08-12 23:11:42,562 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 0 time(s).
2011-08-12 23:11:43,563 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 1 time(s).
2011-08-12 23:11:44,563 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 2 time(s).
2011-08-12 23:11:45,564 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 3 time(s).
2011-08-12 23:11:46,565 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 4 time(s).
2011-08-12 23:11:47,566 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 5 time(s).
2011-08-12 23:11:48,568 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 6 time(s).
2011-08-12 23:11:49,569 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 7 time(s).
2011-08-12 23:11:50,375 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Need another 2 map output(s) where 0 is already in progress
2011-08-12 23:11:50,376 INFO org.apache.hadoop.mapred.ReduceTask: attempt_201108121808_0005_r_000000_2 Scheduled 0 outputs (1 slow hosts and0 dup hosts)
2011-08-12 23:11:50,376 INFO org.apache.hadoop.mapred.ReduceTask: Penalized(slow) Hosts: 
2011-08-12 23:11:50,376 INFO org.apache.hadoop.mapred.ReduceTask: nothing Will be considered after: 49 seconds.
2011-08-12 23:11:50,569 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 8 time(s).
2011-08-12 23:11:51,570 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 9 time(s).
2011-08-12 23:11:51,571 FATAL org.apache.hadoop.mapred.Task: attempt_201108121808_0005_r_000000_2 GetMapEventsThread Ignoring exception : java.net.ConnectException: Call to /127.0.0.1:49442 failed on connection exception: java.net.ConnectException: Connection refused
	at org.apache.hadoop.ipc.Client.wrapException(Client.java:1134)
	at org.apache.hadoop.ipc.Client.call(Client.java:1110)
	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:226)
	at $Proxy0.getMapCompletionEvents(Unknown Source)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$GetMapEventsThread.getMapCompletionEvents(ReduceTask.java:2780)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$GetMapEventsThread.run(ReduceTask.java:2744)
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:567)
	at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:206)
	at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:408)
	at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:425)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:532)
	at org.apache.hadoop.ipc.Client$Connection.access$2300(Client.java:210)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1247)
	at org.apache.hadoop.ipc.Client.call(Client.java:1078)
	... 4 more

2011-08-12 23:11:52,571 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 0 time(s).
2011-08-12 23:11:53,572 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 1 time(s).
2011-08-12 23:11:54,573 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 2 time(s).
2011-08-12 23:11:55,574 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 3 time(s).
2011-08-12 23:11:56,574 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 4 time(s).
2011-08-12 23:11:57,575 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 5 time(s).
2011-08-12 23:11:58,580 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 6 time(s).
2011-08-12 23:11:59,581 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 7 time(s).
2011-08-12 23:12:00,581 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 8 time(s).
2011-08-12 23:12:01,585 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 9 time(s).
2011-08-12 23:12:01,586 INFO org.apache.hadoop.mapred.Task: Communication exception: java.net.ConnectException: Call to /127.0.0.1:49442 failed on connection exception: java.net.ConnectException: Connection refused
	at org.apache.hadoop.ipc.Client.wrapException(Client.java:1134)
	at org.apache.hadoop.ipc.Client.call(Client.java:1110)
	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:226)
	at $Proxy0.statusUpdate(Unknown Source)
	at org.apache.hadoop.mapred.Task$TaskReporter.run(Task.java:635)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:567)
	at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:206)
	at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:408)
	at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:425)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:532)
	at org.apache.hadoop.ipc.Client$Connection.access$2300(Client.java:210)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1247)
	at org.apache.hadoop.ipc.Client.call(Client.java:1078)
	... 4 more

2011-08-12 23:12:02,619 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 0 time(s).
2011-08-12 23:12:03,622 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 1 time(s).
2011-08-12 23:12:04,623 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 2 time(s).
2011-08-12 23:12:05,624 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 3 time(s).
2011-08-12 23:12:06,624 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 4 time(s).
2011-08-12 23:12:07,625 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 5 time(s).
2011-08-12 23:12:08,626 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 6 time(s).
2011-08-12 23:12:09,627 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 7 time(s).
2011-08-12 23:12:10,627 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 8 time(s).
2011-08-12 23:12:11,628 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: /127.0.0.1:49442. Already tried 9 time(s).
2011-08-12 23:12:11,631 FATAL org.apache.hadoop.mapred.Task: Failed to contact the tasktracker
java.net.ConnectException: Call to /127.0.0.1:49442 failed on connection exception: java.net.ConnectException: Connection refused
	at org.apache.hadoop.ipc.Client.wrapException(Client.java:1134)
	at org.apache.hadoop.ipc.Client.call(Client.java:1110)
	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:226)
	at $Proxy0.fatalError(Unknown Source)
	at org.apache.hadoop.mapred.Task.reportFatalError(Task.java:286)
	at org.apache.hadoop.mapred.ReduceTask$ReduceCopier$GetMapEventsThread.run(ReduceTask.java:2763)
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:567)
	at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:206)
	at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:408)
	at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:425)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:532)
	at org.apache.hadoop.ipc.Client$Connection.access$2300(Client.java:210)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1247)
	at org.apache.hadoop.ipc.Client.call(Client.java:1078)
	... 4 more
